_targets.R

```diff
@@ -249,7 +249,7 @@ list(
   ),
   tar_target(
     name = params_id,
-    command = get_params(param_file_path_id),
+    command = get_params(param_file_path_id) |> as.data.frame(),
```

Comment:
Are these intentional changes? I thought this part of the pipeline been refactored to expect the raw output of `get_params`, but perhaps not

Reply:
Oh no I think these are changes that @gvegayon had made initially, to get things to work, but now are not needed. I will revert (though I don't think we really need to change anything in `_targets.R` since it is not the eval pipeline but the old production pipeline).